WooCommerce/Classes/ViewRelated/Products/Edit Product/Cells/ImageAndTitleAndTextTableViewCell.swift

```diff
@@ -129,6 +142,24 @@ extension ImageAndTitleAndTextTableViewCell {
         titleLabel.numberOfLines = 0
         contentView.backgroundColor = .warningBackground
     }
+
+    func updateUI(topLeftImageViewModel: TopLeftImageViewModel) {
+        let viewModel = ViewModel(title: topLeftImageViewModel.title,
+                                  text: nil,
+                                  image: topLeftImageViewModel.icon,
+                                  imageTintColor: topLeftImageViewModel.iconColor,
+                                  numberOfLinesForTitle: 0,
+                                  isActionable: false)
+        updateUI(viewModel: viewModel)
+
+        if topLeftImageViewModel.isFootnoteStyle {
+            titleLabel.applyFootnoteStyle()
+        } else {
+            titleLabel.applyBodyStyle()
+        }
+
+        contentImageViewWidthConstraint.isActive = true
```

Comment:
My motivation here was to align the text that is following after the image. That's why I fixed the size of the image view. The content mode for the image view was already set to `center`, so if the image is smaller, it will be the right size and centered within the image view.
For example, disabling that constraint produces the following result on the order screen:
<img width="504" alt="Screenshot 2020-12-29 at 09 21 21" src="https://user-images.githubusercontent.com/54751/103271693-3281a000-49bb-11eb-8068-e88c71082da2.png">

I couldn't think of a generic way to fix this without fixing the image size (or maybe text left margin).

Reply:
Got it, thanks for the explanation! Yea, it's tricky to align images with different width. Let's go with fixed width here, and I'll see how it looks after replacing `TopLeftImageTableViewCell` with `ImageAndTitleAndTextTableViewCell` later 👍